dist/images/ovnkube.sh

```diff
@@ -934,6 +942,7 @@ ovn-master() {
     ${egressip_enabled_flag} \
     ${egressfirewall_enabled_flag} \
     --metrics-bind-address ${ovnkube_master_metrics_bind_address} \
+    ${egress_interface} \
```

Comment:
why do we need it on the master?

Reply:
That's right, we can do everything with the node parameter. I'll change that in the other pr